src/StarkEx.Crypto.SDK/Signing/IStarkExSigner.cs

```diff
@@ -4,6 +4,10 @@
 
 /// <summary>
 /// An interface for generating and verifying signatures on a STARK-friendly curve.
+/// Implementation is based on StarkWare's reference implementation for StarkEx cryptographic operations.
```

Comment:
nitpick: This is the interface and there is no implementation. Should this comment be in the class? Or is the intent to say something like: "Implementations should follow the reference implementation from StarkWare"?

Reply:
Can we also add the link to the file with the implementation directly?

I believe it is https://github.com/starkware-libs/starkware-crypto-utils/blob/dev/src/js/key_derivation.js

Reply:
Changed